README.md

```diff
@@ -45,20 +48,21 @@ instructions under "Running Old Versions" to display the old guides.
 After cloning this repo, install dependencies and run the demo app:
 
 ```bash
-$ npm install
-$ ember serve
+npm install
+bower install
```

Comment:
Yeah, if I do `bower install && ember serve`, I get:
```
Path must be a string. Received null
TypeError: Path must be a string. Received null
    at assertPath (path.js:7:11)
    at Object.join (path.js:1213:7)
    at CoreObject.<anonymous> (/opt/homebrew/lib/node_modules/ember-cli/lib/models/addon-discovery.js:117:26)
    at Array.map (native)
    at CoreObject.extend.discoverFromDependencies (/opt/homebrew/lib/node_modules/ember-cli/lib/models/addon-discovery.js:107:70)
    at CoreObject.extend.discoverChildAddons (/opt/homebrew/lib/node_modules/ember-cli/lib/models/addon-discovery.js:69:33)
    at CoreObject.extend.discoverAddons (/opt/homebrew/lib/node_modules/ember-cli/lib/models/addon.js:212:42)
    at CoreObject.extend.initializeAddons (/opt/homebrew/lib/node_modules/ember-cli/lib/models/addon.js:225:10)
    at setupRegistryForEachAddon (/opt/homebrew/lib/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:18:10)
    at Object.module.exports.setupRegistry (/opt/homebrew/lib/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:46:3)
```

Obviously `npm install && ember serve` also fails; if it worked, then this PR wouldn't exist. :)
But for completeness, it produces this error:

```
Missing bower packages: 
Package: font-awesome
  * Specified: ~3.2.1
  * Installed: (not installed)

Package: ember-qunit-builds
  * Specified: ~0.4.0
  * Installed: (not installed)

Package: d3
  * Specified: ~3.5.3
  * Installed: (not installed)

[MR - snip snip]

Run `bower install` to install missing dependencies.
```